internal/venice-consumer/src/main/java/com/linkedin/venice/client/consumer/VeniceChangelogConsumerClientFactory.java

```diff
@@ -40,11 +40,16 @@ public synchronized <K, V> VeniceChangelogConsumer<K, V> getChangelogConsumer(St
       newStoreChangelogClientConfig
           .setSchemaReader(ClientFactory.getSchemaReader(newStoreChangelogClientConfig.getInnerClientConfig()));
 
-      String viewClass = getViewClass(
-          storeName,
-          newStoreChangelogClientConfig.getViewClassName(),
-          d2ControllerClient,
-          globalChangelogClientConfig.getControllerRequestRetryCount());
+      String viewClass = newStoreChangelogClientConfig.getViewClassName() == null
+          ? ""
+          : newStoreChangelogClientConfig.getViewClassName();
+      if (!viewClass.isEmpty()) {
+        viewClass = getViewClass(
+            storeName,
+            newStoreChangelogClientConfig.getViewClassName(),
```

Comment:
Just another minor thing: we might need to change config field name: `newStoreChangelogClientConfig.getViewClassName()` to `newStoreChangelogClientConfig.getViewName()`. It is not a class name anymore?